docs/dev/README.md

```diff
@@ -5,91 +5,27 @@ operator projects to see if they pass Red Hat operator certification
 requirements.
 
 The project will include a commandline interface that will accept your operator
-bundle image as an input, and will run validate that your operator bundle
-complies with a configurable set of checks.
+bundle or container image as an input, and will run validate that your container
+image or operator bundle complies with a series of validations.
```

Comment:
is validation a synonym of check at the end of Line 9?  I think we should be consistent with out terms and point out if we are talking about something different than checks.